### PR TITLE
Clean plotfile output for btd

### PR DESCRIPTION
**Pull request by RevathiJambunathan**
In this PR, the path to the plotfile output for BTD is fixed.
i.e., `diags/<diag_name>0000*`

To be merged after #1717

### CODE REVIEW

File: Source/Diagnostics/WarpXOpenPMD.cpp [hunk 1]
```diff
@@ -287,8 +287,8 @@ WarpXOpenPMDPlot::Init (openPMD::Access access, const std::string& filePrefix, b
     if (amrex::ParallelDescriptor::NProcs() > 1) {
 #if defined(AMREX_USE_MPI)
         m_Series = std::make_unique<openPMD::Series>(
-            filename, access,
-            amrex::ParallelDescriptor::Communicator()
+                filename, access,
+                amrex::ParallelDescriptor::Communicator()
```
Comment (RevathiJambunathan) [inline Source/Diagnostics/WarpXOpenPMD.cpp]:
```suggestion
            filename, access,
            amrex::ParallelDescriptor::Communicator()
```

File: Source/Diagnostics/BTDiagnostics.cpp [hunk 2]
```diff
@@ -889,3 +885,4 @@ BTDiagnostics::InterleaveFabArrayHeader(std::string Buffer_FabHeader_path,
     snapshot_FabHeader.WriteMultiFabHeader();
 
 }
+
```
Comment (RevathiJambunathan) [inline Source/Diagnostics/BTDiagnostics.cpp]:
```suggestion
```

**Review by atmyers — APPROVED** [review]

**Comment by ax3l** [comment]
Restarting CI to make sure coverage is on

**Review by ax3l — APPROVED** [review]
:tada:

**Comment by ax3l** [comment]
@RevathiJambunathan can you please rebase against  the latest `development` (or alternatively merge the latest `development`)? We fixed some CI timeout issues yesterday.

**Comment by RevathiJambunathan** [comment]
@ax3l I rebased with the current development branch. But, the DPC++ ubuntu build fails with an error
```
/usr/bin/ld: final link failed: No space left on device
LLVM ERROR: IO failure on output stream: No space left on device
PLEASE submit a bug report to https://software.intel.com/en-us/support/priority-support and include the crash backtrace.

Any suggestions?
```

**Comment by ax3l** [comment]
Thanks for the ping, @RevathiJambunathan. Looks like DPC++ and MKL got too large for CI. I disabled them temporarily now and will try to get it back online in #1743.

**Pull request merged**